platforms/BUILD

```diff
@@ -74,4 +74,5 @@ platform(
         "EXECUTOR": "cortex_m7_fpu",
     },
     parents = [":cortex_m"],
+    visibility = ['//visibility:public'],
```

Comment:
I am happy to change the visibility on all the platforms. What is the problem that you are trying to solve with the public visibility?

Reply:
With the current visibility, I can run the following:

`bazel build --platforms=@bazel-embedded/cortex_m7_fpu`

However I cannot do the following:

`bazel build --platforms=//platforms/custom_board`

Assuming the I defined the following platform (in platforms/BUILD):

`
platform(
      name = "custom_board",
      parents = ["@bazel_embedded//platforms:cortex_m7_fpu"],
+
`

With visibiity public, this works.

Reply:
I have opened another PR with the visibility changes (#16).

Reply:
True that makes sense